driver/src/models/standing_order.rs

```diff
@@ -17,6 +21,24 @@ impl StandingOrder {
     }
 }
 
+impl ConcatingHashable for Vec<StandingOrder> {
+    fn concating_hash(&self, init_hash: H256) -> H256 {
+       let mut hasher = Sha256::new();
+        hasher.input(init_hash);
+        for i in 0..models::NUM_RESERVED_ACCOUNTS {
```

Comment:
Same argument about efficiency as in the other PR applies. I'm starting to think it would be easier if the db_interface always returned a vector of 50 elements with 0 orders for the ones that don't have any standing orders at the moment.

But that could also be done in another PR.

Reply:
Made an issue